tests/test_settings.py

```diff
@@ -33,6 +33,12 @@ def test_parameter_containing_underscore_with_an_environment_variable(elasticsea
         assert '500' == thread_pool_queue_size
 
 
+def test_setting_processors(elasticsearch):
+    # The fixture for this test comes from tests/docker-compose.yml
+    for processors in elasticsearch.get_processors_setting():
```

Comment:
This call is to `get_processors_setting` but the new function is defined as `get_processors_settings` (extra `s` on the end).

Reply:
Sorry for that @jarpy! I pushed ddb3897698ac487340c006d8057c8da175791a5e.